tests/auth/test_auth_store.py

```diff
@@ -305,3 +305,24 @@ async def test_loading_does_not_write_right_away(
     # Once for the task
     await hass.async_block_till_done()
     assert hass_storage[auth_store.STORAGE_KEY] != {}
+
+
```

Comment:
Coverage was quite light here so I did a full cycle of add/check tokens/remove/check tokens